### PR TITLE
Transfer graphing from BioMass to Text2Model

### PR DESCRIPTION
**Pull request by formersbach**
This pull request transfers the graphing capabilities from the BioMass level to the Text2Model level.

### CODE REVIEW

**Comment by formersbach** [comment]
Hey @himoto 
Please review whether I correctly changed `model_object.py`  especially careful!

File: biomass/construction/text2model.py [hunk 1]
```diff
@@ -4,6 +4,9 @@
 from dataclasses import dataclass, field
 from typing import Dict, Final, List, Literal, Optional
 
+import numpy as np
+import pygraphviz as pgv
+from pyvis.network import Network
```
Comment (himoto) [inline biomass/construction/text2model.py:9]: Currently biomass should be able to use without installing `pygraphviz` and `pyvis` (see #162).
Can you please move these imports from top-level to inside-function?
Also, remember to write:
```python
try:
    import pygraphviz as pgv
except ImportError:
    print("pygraphviz is required to run this function.")
```
in the beginning of `graph()` property and 
```python
try:
    from pyvis.network import Network
except ImportError:
    print("pyvis is required to run this function.")
```
for `dynamic_plot()` function.

**Review by himoto — CHANGES_REQUESTED** [review]
Thanks for your PR, looks so nice to me!
Before merging into master, I have a small request.
Please refer to my comments.

**Comment by himoto** [comment]
Hi @formersbach, can you please update .pre-commit-config.yaml file?
https://github.com/biomass-dev/biomass/blob/2e2bcd783806748d64f27e9563ffded14c3e3485/.pre-commit-config.yaml#L13
In line 13, please bump python version from 3.7 to 3.8 in order to support walrus operator (:=).

**Comment by himoto** [comment]
**Question:**
Does your updates affect [`test_graph.py`](https://github.com/biomass-dev/biomass/blob/master/tests/test_graph.py)?
If yes, please also update the test code to be compatible the latest graphing method.

File: biomass/construction/text2model.py (hunk 1 above)
Comment (formersbach) [reply biomass/construction/text2model.py:9]: Woops, good call. Thanks for noticing. 
Should I also do the same with future features such as the steady state analysis?

Comment (himoto) [reply biomass/construction/text2model.py]: We can discuss when you raise an issue. Please let me know new dependencies (packages) in the new feature.
Regarding `the steady state analysis', in my opinion it is ok to add `sympy` to requirements.txt

**Review by himoto — APPROVED** [review]

**Comment by himoto** [comment]
Thanks @formersbach! I will soon merge these changes into master.

**Pull request merged**